packages/components/scripts/cleanReadmes.ts

```diff
@@ -22,7 +22,7 @@ const adjustHeadline = (str: string): string => {
   const h1Replacements: { [key in TagName]?: string } = {
     'p-flex': 'Flex',
     'p-grid': 'Grid',
-    'p-headline': 'Typography',
+    'p-heading': 'Typography',
```

Comment:
I think the table of contents is missing `p-headline` here: https://designsystem.porsche.com/issue/2130/components/typography/props

In addition the possible values for `variant` don't seem to be correct.